plugin-building/build.gradle

```diff
@@ -17,6 +17,14 @@ android {
   lintOptions {
     abortOnError false
   }
+
+  testOptions {
+    unitTests.all {
+      jacoco {
+        includeNoLocationClasses = true
+      }
+    }
+  }
```

Comment:
is there a way we can declare the configuration once and use it for all plugins?

Reply:
Previously I had it directly in the `jacoco.gradle` file but since it requires to be inside `android` I found it only really works if it's in each modules build.gradle.